spyre-rag/ui/src/components/customSendMessage.jsx

```diff
@@ -170,7 +196,12 @@ async function customSendMessage(request, _options, instance) {
     });
 
   } catch (err) {
-    console.error("Error during streaming:", err);
+
+    instance.updateIsLoadingCounter('decrease');
+
+    let errorText = "Sorry, something went wrong during streaming.";
```

Comment:
If we overwrite this variable with `err.error` immediately after this statement, is the initial assignment necessary? Also, `Something went wrong` is a bit vague, but at the same time, are we sure we want to expose raw internal error messages directly to the user via err.error?

Reply:
no need to tell sorry, we can keep the error message straight forward text, may be something like: `Error occurred during active stream`